src/components/Address.vue

```diff
@@ -21,71 +19,74 @@
                 </router-link>
             </div>
         </section>
+
+        <section v-if="onSelectAddress">  
+        <div>
+            <span> {{ onSelectAddress.rua }}</span> 
+            <span> {{ onSelectAddress.bairro }}</span> 
+            <span> {{ onSelectAddress.numero }}</span>
+            <span> {{ onSelectAddress.complemento }}</span>
+        </div>
+        </section>
+
     </div>
 </template>
 
 <script>
     import { addresses } from "../utils/addresses"
     export default {
         data() {
-            return {
-                addresses
+            return { 
+                addresses: addresses
```

Comment:
Dica: quando o nome da váriavel for do mesmo nome que o nome da chave do objeto, pode ocultar.  Pra ficar mais enxuto.